.travis.yml

```diff
@@ -13,5 +13,9 @@ before_script:
   - php artisan key:generate
   - php artisan migrate --database=testing
 
-script:
-  - vendor/bin/phpunit
+jobs:
+  include:
+    - stage: test
+      script: vendor/bin/phpunit
+    - stage: test
+      script: vendor/bin/phpcs app config
```

Comment:
You can also specify the folders inside phpcs.xml and when you change that, please add all folders (database, resources (lang, views), routes and tests)

Reply:
Good point, will do that in the refactor of this PR.